README.md

```diff
@@ -4,19 +4,27 @@ _**New!** We've replaced our documentation with a "For You" feed! As always, [pl
 
 # For You
 
-> **loglot** updated the For You feed! 🔼
-
-> **mybearworld** Improved the VSCode experience! 🐻
-
-> **mybearworld** just contributed to the For You feed! 🐻
+> **mybearworld** ~~improved~~ worsened* the VSCode experience! 🐻
 
 > **TodePond** just added a For You feed! ⭐
 
+> **WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite** Now has monetization! 🎉
+
 > **TodePond** just merged a PR! 🚀
 
+> **loglot** fixed a change that **mybearworld** made to VSCode 
```

Comment:
_Excuse me?!_ You have severely worsened it! Even if you disagree, this is not even in the scope of this PR.

```suggestion
```

Reply:
... no, noone likes a bright code editor, this is beyond the joke nature of this language, this is serious,

Reply:
Wdym?

Reply:
I mean the original comment you're likely going to delete after I posted this comment:

> ... no, noone likes a bright code editor, this is beyond the joke nature of this language, this is serious,

Reply:
You're just putting words in my mouth

Reply:
i said

> ... no, no one likes a bright code editor, this is serious,